tests/data/desc-001.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--<!DOCTYPE document PUBLIC "+//IDN docutils.sourceforge.net//DTD Docutils Generic//EN//XML" "http://docutils.sourceforge.net/docs/ref/docutils.dtd">-->
+
+<document source="desc-001.rst">
+    <section ids="test-section" names="test \section">
+        <title>Test Section</title>
+        <paragraph>This a test section.</paragraph>
+        <desc desctype="attribute" domain="py" noindex="False" objtype="attribute">
```

Comment:
It would be nice if you can provide different `objtype`s. One `<desc>` for `attribute`, another one for `function` and so on.